internal/receiver/scriptreceiver/README.md

```diff
@@ -0,0 +1,84 @@
+# Script Receiver
+
+<!-- status autogenerated section -->
+| Status        |                        |
+| ------------- |------------------------|
+| Stability     | [in-development]: logs |
+| Distributions | [contrib]              |
+
+[in-development]: https://github.com/open-telemetry/opentelemetry-collector#in-development
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+<!-- end autogenerated section -->
+
+## Overview
+Script receiver is a component that performs log collection equivalent to what the UF does when Unix and Linux Technical Add-on is installed.
+
+
+| Script name           | Description                                                                                                                                                                                                                  |
```

Comment:
I think having this table after the config block as valid options could be helpful in introducing the component.